eventmesh-dashboard-console/src/main/java/org/apache/eventmesh/dashboard/console/function/client/ClientTypeEnum.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.dashboard.console.function.client;
+
+public enum ClientTypeEnum {
+
+    RUNTIME,
+
+    STORAGE_ROCKETMQ_REMOTING,
+
+    STORAGE_ROCKETMQ_PRODUCER,
+
+    STORAGE_ROCKETMQ_CONSUMER,
+
+    STORAGE_REDIS,
+
+    CENTER_NACOS,
+    CENTER_NACOS_CONFIG,
+
+    CENTER_NACOS_NAMING,
+
```

Comment:
Please use `Meta` instead of `Center`. Please check to see if there is anything missing in the other code that hasn't been changed.